addons/binding/org.openhab.binding.coolmasternet/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,133 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="coolmasternet" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<bridge-type id="controller">
+		<label>CoolMasterNet Controller</label>
+		<description>A CoolMasterNet Controller (connected to one or more HVAC systems)</description>
+
+		<config-description>
+			<parameter name="host" type="text" required="true">
+				<label>CoolMasterNet Network Hostname or IP address</label>
```

Comment:
Try to keep the label short - usually we simply put "Hostname" or "IP address" here.